x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step.tsx

```diff
@@ -30,7 +35,12 @@ export const ConfigurationStep: FC<CreateAnalyticsStepProps> = ({
   return (
     <EuiForm className="mlDataFrameAnalyticsCreateForm" data-test-subj={dataTestSubj}>
       {showForm && (
-        <ConfigurationStepForm actions={actions} state={state} setCurrentStep={setCurrentStep} />
+        <ConfigurationStepForm
+          actions={actions}
+          isClone={isClone}
```

Comment:
When I clone a job which has used a runtime field as the dependent variable, then make an edit to the runtime field in the form JSON editor, I am seeing issues with the includes table in the form, and the includes section in the advanced JSON editor, where the runtime field isn't being selected properly.

Here the dependent variable isn't checked in the table:

![image](https://user-images.githubusercontent.com/7405507/113161123-ff5ae100-9235-11eb-96de-0f0cd205984b.png)

and here it says it isn't included, even after manually adding it to the `includes` section:

![image](https://user-images.githubusercontent.com/7405507/113161265-1ef20980-9236-11eb-961f-e41fdfc5bd85.png)

Reply:
Thank you for testing, @peteharverson 🙏  Cloning issue with not having the runtime field selected in the table is fixed in c5f3727b7fde7b847004767e23e07c486999edce